modules/validations/incompatible_source.js

```diff
@@ -30,6 +30,12 @@ export function validationIncompatibleSource() {
 
     return entitySources
       .map(source => {
+        try {
+          const sourceURL = new URL(source);
+          source = sourceURL.hostname;
+        } catch (e) {
+          // Source string is not a url
```

Comment:
Looks like this fallthrough will preserve the original passed-in value of `source`. 👍